mockgcp/mockcontainer/cluster.go

```diff
@@ -318,6 +318,14 @@ func (s *ClusterManagerV1) UpdateCluster(ctx context.Context, req *pb.UpdateClus
 		update.DesiredDefaultEnablePrivateNodes = nil
 	}
 
```

Comment:
AGENT: The mock correctly creates a `NetworkConfig` if it's `nil`. This is good defensive programming for the mock's internal state.

Reply:
Thanks!  But maybe we should also tell the agent not to worry as much about mockgcp, our standard for mockgcp should be that it gives the same output as realgcp (as determined by the overnight tests)